### ISSUE TITLE
Block sequence does not affect the order in the sidebar

### ISSUE THREAD

**Issue by ajnyga** [ajnyga/facebookFeed#1]
Hi, tagging @asmecher and @nateWr here in case this is actually an issue in OJS/pkp-lib and not only this plugin.

The plugin adds a new block which you can add to the sidebar. However, for some reason the facebookFeed block is _always_ on top of all other blocks regardless of the sequence you give it.

In the Appearance settings page the block order is displayed correctly. Also if I look at the plugin_settings table, the seq value is what it should be and also in the cache folder the settings cache files have the correct values.

But still, in the actual sidebar, the facebookFeed block is always the first one even when it has for example seq 2 and there is another block with seq 1.

Any idea why?

**Comment by NateWr** [comment]
Is the order respected for other blocks, when you change them?

**Comment by ajnyga** [comment]
@NateWr yes, I have three other block there. One is from the custom block plugin and two basic blocks. I can arrange the three other block without any problems, but the facebook block is always shown on top regardless where I drag it. In the settings the order is correct but in the front page facebook is first. (and yes I do save the settings form after I change the order)

**Comment by asmecher** [comment]
@ajnyga, is this with the master branch, or `ojs-stable-3_1_1`, or one of the releases? I made some recent changes to block loading conventions that might need a change in your plugin too.

**Comment by ajnyga** [comment]
ok, that must be it. I tried to use the plugin in our production which is still in 3.1.0.1. I did change remove the theme plugin template override part which is not supported. Evertyhing else works but there must be some detail that is not supported in 3.1.0.1 left because the sequence seemed to work in a test install with 3.1.1. So I'll just wait till we upgrade to 3.1.1. Sorry for the partly false alarm.
